extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -35,7 +35,7 @@ let PropertyRulesDef;
  */
 const GLOBAL_PROPERTY_RULES = {
   'class': {
-    blacklistedValueRegex: '(^|\\W)i-amphtml-',
+    denylistedValueRegex: '(^|\\W)i-amphtml-',
```

Comment:
/cc @choumx I can't tell if this value is actually exposed to publishers. If so, this isn't safe to change.

Reply:
It's not, this was copied from validator code.

Reply:
It's not safe to change? Or it's not exposed to publishers?

Reply:
Answered offline with @choumx, it's safe to change.